scheduler/test/cook/test/scheduler/scheduler.clj

```diff
@@ -1551,80 +1643,103 @@
                                                                                       :group group-ent-id
                                                                                       :name "job-1"
                                                                                       :ncpus 3
-                                                                                      :memory 2048))
+                                                                                      :memory 2048
+                                                                                      :pool "test-pool"
+                                                                                      ))
```

Comment:
These should be on the previous line (there's a few more below as well)

Reply:
modified all occurrences